cypress/integration/data-import/view-all.spec.js

```diff
@@ -0,0 +1,56 @@
+import ViewAllView from '../../support/fragments/data_import/viewAllView';
+import getRandomPostfix from '../../support/utils/stringTools';
+
+describe('ui-data-import: "View all" log screen', () => {
+  // Create unique file name with given type to upload
+  const fileType = 'mrc';
+  const uniqueFileName = `test${getRandomPostfix()}.${fileType}`;
+
+  before(() => {
+    cy.login(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+    cy.getToken('diku_admin', 'admin');
+
+
+    cy.uploadFile(uniqueFileName);
+    ViewAllView.gotoViewAllPage();
+
+    const query = '((status any "COMMITTED ERROR")) sortby completedDate/sort.descending';
+    cy.getJobLogs({ limit: 1, query });
+  });
+
+  it('C11112 allows to search with "File name" or "ID"', () => {
```

Comment:
need to keep consistency between test in testrail and test in github. You can join tests into gihub or separate tests into testrail

Reply:
or you can define collection of required filters and play the test to each filter